homeassistant/components/light/xiaomi_miio.py

```diff
@@ -204,11 +205,11 @@ def async_turn_on(self, **kwargs):
         """Turn the light on."""
         if ATTR_BRIGHTNESS in kwargs:
             brightness = kwargs[ATTR_BRIGHTNESS]
-            percent_brightness = int(100 * brightness / 255)
+            percent_brightness = int(ceil(100 * brightness / 255.0))
```

Comment:
Do you need int when you have ceil?

Reply:
You are right. python3 returns an int. math.ceil of python2.7 returns a float.